### DISCUSSION TITLE
Add `3.0.3` OpenAPI support

### DISCUSSION THREAD

**Discussion by romalytvynenko** [dedoc/scramble#773] [Ideas]
The tooling out there usually work with this OpenAPI version so it will be helpful to support it.

**Comment by sandervanzutphen** [comment]
+1!
We also would like 3.0.1, per request of our users.

**Comment by Wulfheart** [comment]
![image](https://github.com/dedoc/scramble/assets/25671390/514a7944-ce0a-40eb-b27c-8788c5719e54)
Seems to be supported now. I guess this can be closed then.

**Comment by sandervanzutphen** [comment]
@Wulfheart, no, it does not. 3.1.0 isn't the same as 3.0.1 or 3.0.3

**Comment by Wulfheart** [comment]
@sandervanzutphen thabks for the clarification.
